packages/plus/src/components/OrderSummary/index.tsx

```diff
@@ -0,0 +1,146 @@
+width: 27.5rem;
+border: 1px solid ${({ theme }) => theme.colors.neutral.border};
+`
+
+const HeaderContainer = styled(Stack)`
+height: ${({ theme }) => theme.sizing[900]};
+border-bottom: 1px solid ${({ theme }) => theme.colors.neutral.border};
+padding: ${({ theme }) => theme.space[3]};
+padding-bottom: ${({ theme }) => theme.space[2]};
+`
+
+export const OrderSummary = ({
+  header,
+  hideTimeUnit = false,
+  periodOptions = Units as TimeUnit[],
+  valueUnitInput = 1,
+  unitUnitInput = 'hours',
+  items,
+  locales = estimateCostLocales,
+  currency = 'EUR',
+  localeFormat = 'en-US',
+  footer,
+  children,
+  discount = 0,
+  totalPriceInfo,
+}: OrderSummaryProps) => {
+  const [timePeriodUnit, setTimePeriodUnit] = useState<TimeUnit>(unitUnitInput)
+  const [timePeriodAmount, setTimePeriodAmount] = useState(valueUnitInput)
+
+  const categoriesPrice = useMemo(() => {
+    const listCategories: Record<string, number> = {}
+    items.forEach(
+      category =>
+        (listCategories[category.category] =
+          category.subCategories?.reduce(
+            (acc, subCategory) =>
+              acc +
+              (calculatePrice(
+                subCategory.price ?? 0,
+                subCategory.amount ?? 1,
+                subCategory.amountFree,
+                hideTimeUnit ? 'hours' : timePeriodUnit,
+                hideTimeUnit ? 1 : timePeriodAmount,
+                subCategory.discount,
+                subCategory.fixedPrice,
+              ) || 0),
+            0,
+          ) || 0),
+    )
+
+    return listCategories
+  }, [hideTimeUnit, items, timePeriodAmount, timePeriodUnit])
+
+  const totalPrice = useMemo(() => {
+    const price = Object.values(categoriesPrice).reduce((a, b) => a + b, 0)
+
+    return { before: price, after: Math.max(price * (1 - discount), 0) }
+  }, [categoriesPrice, discount])
+
+  const valueContext = useMemo(
+    () => ({
+      currency,
+      localeFormat,
+      items,
+      categoriesPrice,
+      hideTimeUnit,
+      timePeriodUnit,
+      timePeriodAmount,
+      locales,
+    }),
+    [
+      currency,
+      localeFormat,
+      items,
+      categoriesPrice,
+      hideTimeUnit,
+      timePeriodUnit,
+      timePeriodAmount,
+      locales,
+    ],
+  )
+
+  const computePeriodOptions = useMemo(() => {
+    const computedPeriodOptions: { label: string; value: string }[] = []
+
+    periodOptions.forEach(option =>
+      computedPeriodOptions.push({
+        value: option,
+        label:
+          locales[
+            `estimate.cost.units.${option}.label` as keyof typeof locales
+          ],
+      }),
+    )
+
+    return computedPeriodOptions
+  }, [periodOptions, locales])
+
+  return (
+    <OrderSummaryContext.Provider value={valueContext}>
+      <Container>
+        <HeaderContainer direction="row" justifyContent="space-between">
+          <Text as="p" variant="headingSmallStrong" sentiment="neutral">
+            {header}
+          </Text>
```

Comment:
might be interesting to se it as h3

```suggestion
          <Text as="h3" variant="headingSmallStrong" sentiment="neutral">
            {header}
          </Text>
```